### PR TITLE
Transition fixes

### LINKED ISSUE #670: Accounts screen has jumpy transition
We've worked out most of our transition bugs, and most of our transitions are smooth looking.

One that still isn't (or got bad at some point) is the accounts screen entering view.

When first entering, the entire view is pushed down a little bit, like this:

screen shot 2016-09-14 at 4 12 58 pm

What you can't tell here (but can easily see in practice) is that once it arrives in place, this view jumps up a good bit.

I just did some investigating, and the solution did not seem as obvious to me as I'd hoped it would.  Opening an issue to ensure later investigation.

### PR DESCRIPTION
**Pull request by 2-am-zzz**
Look at that beauty:
- The first-time transitions are a lot better now. A lot better.
- Transitions between the account detail screen and other screens is now smoother (no more of that jumping)

Fixes #670

### CODE REVIEW

**Comment by Zanibas** [comment]
Still requires our tests to pass--not sure what's going on, would like to pair on this.

File: ui/app/accounts/index.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ AccountsScreen.prototype.render = function () {
             style: {
               display: 'flex',
               height: '40px',
-              paddint: '10px',
```
Comment (flyswatter) [inline ui/app/accounts/index.js]: lol how

File: ui/app/actions.js [hunk 2]
```diff
@@ -252,7 +252,6 @@ function createNewVaultAndKeychain (password) {
           return dispatch(actions.displayWarning(err.message))
         }
         dispatch(actions.hideLoadingIndication())
-        forceUpdateMetamaskState(dispatch)
```
Comment (flyswatter) [inline ui/app/actions.js]: ooooo

**Review by danfinlay — APPROVED** [review]
The code looks good.  I want that animation... Ok fine I'm pulling.  This is exciting.

**Review by danfinlay — CHANGES_REQUESTED** [review]
Opened an existing vault for the first time, saw seed word screen with no seed words.

The first time controller is defaulting to first time, even though it's adding new data to the schema.

Need to add a migration to set `firstTime` to false if a vault exists already.

**Comment by Zanibas** [comment]
Closing in favor of breaking these changes into separate parts for better debugging.

**Pull request closed without merging**